compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -308,6 +308,17 @@ def AMDAIELinkExecutables :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIELinkExecutablesPass()";
 }
 
+def AMDAIELoadAlignmentReset :
+    Pass<"iree-amdaie-load-alignment-reset", ""> {
+    let summary = "Reset the alignment of the LLVM load operations.";
+    let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIELoadAlignmentResetPass()";
+    let description = [{
+      Reset the alignment of the LLVM load operations to the 'unset'
+      optional value. This is a workaround for an issue in peano, and
```

Comment:
Have you checked with peano folks on this?

Reply:
Yes, there's a ticket on JIRA (AIECC-589) which they're aware of, and we've discussed a bit on the peano slack channel. Just to note: this pass isn't new/changed -- i've just transplanted it from aie2xclbin.cpp.

Reply:
Yeah, I am aware this existed already, but I find it very strange. Will check the discussions and JIRA to understand more.